projects/plugins/publicize/composer.json

```diff
@@ -1,6 +1,6 @@
 {
-	"name": "automattic/jetpack-reach",
-	"description": "Reach plugin",
+	"name": "automattic/jetpack-publicize",
```

Comment:
I don't think that's going to work since we already have [a Publicize package](https://github.com/Automattic/jetpack-publicize) for the Publicize functionality itself. Maybe this should be called `automattic/jetpack-publicize-plugin` instead, and all references to the Package name should be updated accordingly?